Simplenote/Classes/SignupRemote.swift

```diff
@@ -0,0 +1,49 @@
+import Foundation
+
+// MARK: - SignupRemote
+//
+class SignupRemote {
```

Comment:
Right. I think it need to be discussed, because there are many ways to organise files. Right now those files are in the groups that contain most of the files for that feature. So when you work on onboarding or verification feature you can find all the files in one place :-)

Reply:
Agreed!